.drone.yml

```diff
@@ -1128,6 +1128,103 @@ steps:
     # terminate the node gracefully and wait to allow `valgrind` dump data
     - kill -INT $(cat /var/run/tezedge.pid) && sleep 10
 
+- name: make-in-mem-snapshot-and-resume
+  image: tezedge/tezedge-ci-builder:rust-1.58.1-v12.0-octez
+  user: root
+  pull: if-not-exists
+  volumes:
+    - name: cache
+      path: /cache
+  commands:
+    - rust_libs=$(echo "`rustup show home`/toolchains/`rustup show active-toolchain | tr " " "\n" | head -1`/lib")
+    - export LD_LIBRARY_PATH="drone-cache/build_files/ffi:$rust_libs"
+    - export OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+    - export TEZEDGE_GC_DELAY_SNAPSHOT_SEC="0"
+    - mkdir -p ./target/release && cp drone-cache/build_files/protocol-runner ./target/release
+    - cp -R /cache/tezedge-data ./snapshot-database
+    - rm -r ./snapshot-database/context
+    - >
+      ./drone-cache/build_files/light-node replay --target-path=./replay-snapshot
+      --config-file "./drone-cache/build_files/tezedge/tezedge_drone.config"
+      --identity-file "./drone-cache/build_files/identities/identity_6.json"
+      --protocol-runner=./drone-cache/build_files/protocol-runner
+      --init-sapling-spend-params-file=./drone-cache/build_files/ffi/sapling-spend.params
+      --init-sapling-output-params-file=./drone-cache/build_files/ffi/sapling-output.params
+      --to-block BLR5WXgdQ4vLXxj3rcSG66afrcVQUvwPGxWz2CDu5HAZ7Jo9rNp
+      --tezos-data-dir=./snapshot-database --bootstrap-db-path=bootstrap_db
+      --tezos-context-storage=tezedge --context-kv-store=inmem
+      --network=$${NETWORK} &
+      echo $! > /var/run/tezedge.pid
+    # this command waits that a snapshot is created
+    - |
+      sh -c '
+      block=0
+      attempts=0
+      while [ $attempts -lt 20 ]; do
+        echo "attempts = $attempts"
+        sleep 30
+        if [ -d "./replay-snapshot/context" ]; then
+          echo "found snapshot"
+          break
+        fi
+        attempts=$(($attempts + 1))
+      done
+      '
+    - echo "Bootstrapped. Killing the node and waiting for 10 seconds."
+    # terminate the node gracefully and wait to allow `valgrind` dump data
+    - kill -9 $(cat /var/run/tezedge.pid) && sleep 10
```

Comment:
@sebastiencs is `-9` necessary? it is a bit aggressive, why doesn't SIGINT work?

Reply:
Ah yes, I used `-9` for testing but I forgot to change it back